docs/user/sbt.rst

```diff
@@ -234,9 +234,9 @@ support Scala/JVM or Scala.js if the Native portions have replacement
 code on the respective platforms.
 
 The primary purpose of this feature is to allow libraries to support
-Scala Native that need native `glue` code to operate. The current
+Scala Native that need native "glue" code to operate. The current
```

Comment:
Is this related?

Reply:
Lolgab  thank you for the careful review & the thought it took.

In the spirit of collegial discussion...

Yes, this change and the one immediatly below, takes some explaining.  They are not, IMHO, not extraneous
 "Boy Scout" or, inclusively, "Scout" Rule noise and distraction.

The proposed doc change are related to the base change in the sense that I can not get a clean trial build
of the file I touched.  That is a personal, and probably project requirement for anything I submit.

The doc build  from the current snapshot contains a number (10, 15) Warnings.  At one point, I had submitted
PRs to reduce that to zero.  I believe, for understandable reasons, those PRs are still in the queue.    

This change means that the file I did need to touch does not contribute to the general noise of the doc build.

Worthy enough for inclusion?  Please advise.